app/styles/nypr-member-center.scss

```diff
@@ -0,0 +1,190 @@
+@import "compass/css3/flexbox";
```

Comment:
I'd prefer this file to be broken down into smaller partials that are imported into the main `nypr-account-settings` sass file. there should only be one top left sass file per addon.